packages/firebase_data_connect/firebase_data_connect/example/ios/firebase_app_id_file.json

```diff
@@ -0,0 +1,7 @@
+{
+  "file_generated_by": "FlutterFire CLI",
+  "purpose": "FirebaseAppID & ProjectID for this Firebase app in this directory",
+  "GOOGLE_APP_ID": "1:1033042013828:ios:a56668abf3ea77b62e5cd1",
+  "FIREBASE_PROJECT_ID": "fdc-bugbash-mtewani",
```

Comment:
It doesn't seem to be same project as before

Reply:
Changed to e2e